lib/streamlit/elements/camera_input.py

```diff
@@ -13,35 +13,23 @@
 # limitations under the License.
 
 from dataclasses import dataclass
-from streamlit.type_util import (
-    Key,
-    to_key,
-    LabelVisibility,
-    maybe_raise_label_warnings,
-)
-
 from textwrap import dedent
-from typing import Optional, cast, List, TYPE_CHECKING
-
-from streamlit.proto.CameraInput_pb2 import (
-    CameraInput as CameraInputProto,
-)
+from typing import TYPE_CHECKING, List, Optional, cast
 
+from streamlit.proto.CameraInput_pb2 import CameraInput as CameraInputProto
+from streamlit.runtime.metrics_util import gather_metrics
 from streamlit.runtime.scriptrunner import ScriptRunContext, get_script_run_ctx
 from streamlit.runtime.state import (
-    register_widget,
     WidgetArgs,
     WidgetCallback,
     WidgetKwargs,
-)
-from streamlit.runtime.metrics_util import gather_metrics
-
-from ..proto.Common_pb2 import (
-    FileUploaderState as FileUploaderStateProto,
-    UploadedFileInfo as UploadedFileInfoProto,
+    register_widget,
 )
 from streamlit.runtime.uploaded_file_manager import UploadedFile, UploadedFileRec
+from streamlit.type_util import Key, LabelVisibility, maybe_raise_label_warnings, to_key
 
+from ..proto.Common_pb2 import FileUploaderState as FileUploaderStateProto
```

Comment:
There is no way in isort to automatically resolve relativ imports or?

Reply:
Unfortunately not, but we can convert the imports manually and then add a new pre-commit hook to force absolute imports.
```
      - id: check-no-relative-imports
        language: pygrep
        name: No relative imports
        description: Streamlit style is to use absolute imports only
        entry: "^\\s*from\\s+\\."
        types_or: [python, pyi]
        pass_filenames: true
        files: \.py$
```
What do you think about it?